src/main/java/org/prebid/server/handler/AuctionHandler.java

```diff
@@ -201,16 +201,16 @@ private Usersyncer usersyncerByName(String name) {
         return bidderCatalog.usersyncerByName(adapterNameFor(name));
     }
 
-    private MetaInfo metaInfoByName(String name) {
-        return bidderCatalog.metaInfoByName(adapterNameFor(name));
+    private BidderInfo metaInfoByName(String name) {
```

Comment:
Rename to `bidderInfoByName`.
Pls find all `metaInfoByName` in codebase and do the same (including unit tests).